static/templates/presence_row.hbs

```diff
@@ -5,8 +5,13 @@
           href="{{href}}"
           data-user-id="{{user_id}}"
           data-name="{{name}}">
-            <span class="user-name">{{name}}</span>
-            {{> status_emoji status_emoji_info}}
+            <div>
```

Comment:
This change needs to be made conditional on whether the `with_status` setting is selected.